app/services/data/get-reduction-reasons.js

```diff
@@ -0,0 +1,14 @@
+const config = require('../../../knexfile').web
```

Comment:
Maybe remove the three files which are in the `feature/wmt-823/ref-data-service` PR from this PR? If they are reviewed and updated in the other PR it could lead to conflicts / missed fixes

Also the files from the percentage to absolute hours PR.

We can review all together on the merge branch